fabmukunzi/certificate-manager---Fabrice
# Task1 KAN-7 project setup

#### PR description

- Description of what the PR does
It sets up everything that I will need during the project development including folder structure, webpack, eslint and prettier for formating  

#### Ticket / Jira Link

- [KAN-7](https://task-force-pro-dccs.atlassian.net/browse/KAN-7?atlOrigin=eyJpIjoiYTQ3ZmYyZjJhNjg4NDg4M2E2NmY3ZDkxNzY4ZWM1MWMiLCJwIjoiaiJ9)

#### Checklist:


#### How To Test Changes

- Checkout to this branch `chore/task1-KAN-7-project-setup`
- Type `npm i` to install packages
- Open terminal and type `npm run dev` to run app
- Open your browser and navigate to `http://localhost:3000/` you will see **Hello world** text in a blue color
- In the code editor open **components** folder and navigate to **Button** try to add a semi colon to check if eslint and prettier are working as expected

#### Additional Comments
N/A

#### Screenshots:
N/A
